### DISCUSSION TITLE
Ignore password prompt windows

### DISCUSSION THREAD

**Discussion by canisli** [asmvik/yabai#846] [Q&A]
Is there a way to make it so Yabai ignores windows that prompt for your password? 
<img width="560" alt="image" src="https://user-images.githubusercontent.com/73449574/108584580-54cad680-72f7-11eb-9b4b-ba2fa33774c2.png">

**Comment by feDann** [comment]
Did you find a way to ignore this?

**Reply by azinsharaf** [reply]
when it happens, run the query command to see if it would list that window. If it does and it has unique title you can exclude it with `mange off`.

`yabai -m query --windows`
